xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1310,14 +1394,14 @@ struct AppModel: ModelProtocol {
         state: AppModel,
         environment: AppEnvironment
     ) -> Update<AppModel> {
-        let fx: Fx<AppAction> = environment.data.indexOurSpherePublisher().map({
-            version in
-            AppAction.succeedIndexOurSphere(version: version)
-        }).catch({ error in
-            Just(
-                AppAction.failIndexOurSphere(error.localizedDescription)
-            )
-        }).eraseToAnyPublisher()
+        let fx: Fx<AppAction> = environment.data
+            .indexOurSpherePublisher().map({ receipt in
```

Comment:
Fighting XCode's code formatting bugs. This form of chaining doesn't trigger the indentation bugs.